dashboard/app/views/home/_privacy.md

```diff
@@ -26,7 +26,7 @@ Our Privacy Policy is fueled by our commitment to the following Privacy Principl
 8. We hold our partners and service providers to privacy and security practices no less stringent than our own.
 
 ## Privacy Policy
-We are committed to creating a safe and secure environment for learners of all ages and other visitors on our websites located at [https://code.org](https://code.org/), [https://studio.code.org](https://studio.code.org/), [https://hourofcode.com](https://hourofcode.com/), [https://curriculum.code.org](https://curriculum.code.org/), [https://advocacy.code.org](https://advocacy.code.org/), [https://k12cs.org](https://k12cs.org/), and any other websites or online services (collectively, our “Services”) that link to this Privacy Policy.
+We are committed to creating a safe and secure environment for learners of all ages and other visitors on our websites located at [https://code.org](https://code.org/), [https://studio.code.org](https://studio.code.org/), [https://hourofcode.com](https://hourofcode.com/), [https://curriculum.code.org](https://curriculum.code.org/), [https://advocacy.code.org](https://advocacy.code.org/), [https://k12cs.org](https://k12cs.org/), https<area>://codeprojects.org, and any other websites or online services (collectively, our “Services”) that link to this Privacy Policy.
```

Comment:
I'm guessing the `<area>` is a mistake?
```suggestion
We are committed to creating a safe and secure environment for learners of all ages and other visitors on our websites located at [https://code.org](https://code.org/), [https://studio.code.org](https://studio.code.org/), [https://hourofcode.com](https://hourofcode.com/), [https://curriculum.code.org](https://curriculum.code.org/), [https://advocacy.code.org](https://advocacy.code.org/), [https://k12cs.org](https://k12cs.org/), [https://codeprojects.org](https://codeprojects.org), and any other websites or online services (collectively, our “Services”) that link to this Privacy Policy.
```

Reply:
No, the <area> tag breaks the URL detectability as per [this StackOverflow post](https://stackoverflow.com/questions/25706012/how-do-i-prevent-auto-generated-links-in-the-github-wiki). codeprojects.org doesn't actually resolve, it needs to be a specific student project path within that url for it to resolve. So a hyperlink doesn't make much sense in this context

Reply:
makes sense